docs/README.md

```
@@ -19,111 +19,120 @@ layout:
 
 <div data-full-width="false"><figure><picture><source srcset=".gitbook/assets/hypster_text_white_text.png" media="(prefers-color-scheme: dark)"><img src=".gitbook/assets/hypster_with_text (1).png" alt=""></picture><figcaption></figcaption></figure></div>
 
-### Hypster is a lightweight configuration framework for managing and **optimizing AI & ML workflows**
+### Hypster is a lightweight configuration framework for building, exploring, replaying, and optimizing Python workflows.
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)